### DISCUSSION TITLE
Equipo 8 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#15] [Proyecto Final]
Hola @CaroJC0329  @Juliaarias22 
Aquí está el hilo en el foro para su equipo. Podrían conversar aquí sobre sus avances con el proyecto y avisar el Equipo de Educación de Patrones Hermosos sobre cualquier pregunta que tengan al respecto.
Con la emoción de seguir el desarrollo de su proyecto,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @CaroJC0329 @Juliaarias22

Gracias por entregar su etapa 0 del proyecto final. Estoy revisándola y escribo a ustedes por este medio con seguimiento. Suena muy interesante el foco de su proyecto, es un asunto muy de interés hoy en día.

Al leer la lista de preguntas, ¿parece que eligieron a una entrevistada? ¿Sería alguien que trabaja en una organización ambiental? Pregunto porque las preguntas asumen que la persona trabaja en cierto lugar y quisiera tener el contexto. 

Algunas sugerencias: me parece que sería una entrevista en la cual tendrían que adaptarse a las respuestas que están recibiendo. Entonces, sería súper hacer algunas preguntas de seguimiento (que probablemente no son en su protocolo). También recomiendo empezar con una pregunta (o historia, o algo) para romper el hielo. Y sería importante avisar a la entrevistado/a la duración de la entrevista _antes_ de empezarla, para que la persona esté preparada.

¿Tienen preguntas para mí? Cualquier cosa, aquí ando.

¡Mucha suerte!
Saludos
Kasturi

**Comment by CaroJC0329** [comment]
Buenas noches @KasturiSShah, nos puede conceder una prorroga para el modulo 1.

**Reply by KasturiSShah** [reply]
Hola @CaroJC0329 @Juliaarias22 
Claro que sí. Tienen una prórroga hasta este viernes para completar la etapa 1 el proyecto final.
Saludos,
Kasturi

**Comment by Juliaarias22** [comment]
Buenas noches Kasturi,De antemano una disculpa por la hora, nuevamente solicitamos una prórroga a la etapa 2 de proyecto, la que se entrega hoy.Por favor, muchas gracias.

**Reply by KasturiSShah** [reply]
Hola @Juliaarias22 @CaroJC0329 
Gracias por avisar, y no se preocupen. ¿Qué tal una prórroga hasta este domingo? Mi recomendación sería entregarla lo antes posible, para que puedan usar las próximas dos semanas para crear su prototipado y entregarlo a tiempo (antes de que empiece la Semana Santa, para que no tengan que trabajar durante las vacaciones 😄 ). 
Saludos
Kasturi

**Comment by Juliaarias22** [comment]
Hola Kasturi!! Buenas tardes. Intenté subir la etapa 2 del proyecto a github y no me dejó, la envío por este medio.Link del video: https://www.canva.com/design/DAFeVxD5OMM/0f0zBqOjjkvb2sFjop19vw/watch?utm_content=DAFeVxD5OMM&utm_campaign=designshare&utm_medium=link&utm_source=publishsharelink—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by Juliaarias22** [comment]
Kasturi buenas noches, envio por este medio la etapa 2 del proyecto final ya que github no me permite subirla…

Link de video: https://drive.google.com/file/d/1iNBhaL30iyGjyR5OaWasGB__TtCG_lbj/view?usp=drivesdk

Gracias!!

**Reply by KasturiSShah** [reply]
Hola @Juliaarias22 
Gracias por avisarme. Sí, no es posible subir vídeos a Github. ¿Podrías rellenar este formulario con la liga que pusiste en tu mensaje anterior (y contestar a los otros puntos)? La entrega pide por la liga, no pide que subas el video.
Me avisas como andas. 
Saludos
Kasturi

**Comment by Juliaarias22** [comment]
@KasturiSShah 
Hola Kasturi!! espero qu te encuentres bien, 
solicito una prórroga para la etapa 3, de favor. 
gracias!!

**Reply by KasturiSShah** [reply]
Hola @Juliaarias22 
Todo bien gracias, espero estés bien. Sobre la prórroga, claro, por favor mira mi retroalimentación sobre la etapa 2 a continuación donde contesté a tu solicitud para una prórroga.
Espero pasaste lindo los días feriados.
Kasturi

**Comment by KasturiSShah** [comment]
Hola @Juliaarias22 

Gracias por entregar la etapa 2 del proyecto final. La he revisado y te escribo con retroalimentación. Me gustó el vídeo y la explicación clara de las ideas generadas por 8S y tu uso de SCAMPER para reducir y organizar las ideas. ¡Super!

Algunas sugerencias:
1. Ahora que estás trabajando con la etapa 3 del proyecto final, ya es momento de pensar en el prototipado low fidelity. ¿Cuál sería? Se me ocure varias ideas, como una mapa para cada componente del sistema de agua (plantas tratadoras, tubería, floculantes, colar etc), o una diagrama de flujo mostrando los pasos (floculantes, colar el agua, que hay que hacer cuando el agua esté "limpia", etc), o un flyer que comunica a la gente cómo eliminar contaminantes de agua y la importancia de no trabajar con quimicos tóxicos. Solo son sugerencias, no necesitas usarlas.
2. De hecho, creo que puedes agrupar las ideas un poquito más. Por ejemplo, ideas relacionadas con el proceso de tratamiento del agua y ideas relacionadas con evitar contaminación del agua.

Quisiera aclarar algo: en la etapa entregada, veo que la lista de los miembros del equipo solo tiene tu nombre. Empezando con esta etapa, ¿estás completando el proyecto sola? Si es así, ¿comunicaste con Carolina al respecto? Pregunto para ponerme al corriente con las decisiones del equipo 8 para que puedo actualizar nuestro lista de equipos y facilitar conversaciones necesarias.

Sobre la prórroga para la etapa 3, claro que sí, es posible recibirla. ¿Hasta qué fecha quieres la prórroga?

Quedo atenta cualquier cosita.

Adelante y mucha suerte,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @Juliaarias22 

¿Cómo andas con tu proyecto final? ¿necesitan una prórroga para la etapa 3, 4 o 5 del proyecto final?

¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar tu presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si te conviene la asignación de tu equipo a una de las dos sesiones de clausura del curso, o si quieres cambiar tu día asignado al otro día.

Mi recomendación sería poner 1-2 días de trabajo en completar la etapa 3, 4 y 5, y luego preparar para la presentación final. La etapa 5 no necesita que la etapa 3 ni 4 sea completada. Así que una opción sería pasar ~ 30 minutos completando la etapa 5, entregarla, y luego pasar a la etapa 3 y 4 que necesitan un poquito más trabajo. Depende de ti cómo quieres avanzar, solo pongo la sugerencia aquí por si te sirve.

Estoy aquí para apoyar, déjame saber cualquier cosa.

Mucha suerte, mucho éxito,
Kasturi

**Reply by Juliaarias22** [reply]
Hola @KasturiSShah!!
Justamente estoy trabajando en este momento con la etapa 3! de antemano mil disculpas, he tenido unos proyectos en la escuela y exámenes y no me daba tiempo. También batallé en encontrar mi low-fidelity, estaba bloqueda de ideas pero ya lo tengo solo me queda grabar el video...
gracias Kasturi.

**Reply by KasturiSShah** [reply]
Hola @Juliaarias22 
No te preocupes, espero que todo salió bien con los examenes y proyectos. Me avisas cuando hayas subido la etapa 3 a Github. (Mi recomendación sería no pasar mucho tiempo en la grabación del vídeo, complétalo y luego pasa a las etapas 4 y 5.)
Saludos,
Kasturi
